example_envs/bookinfo/leafnode.rs

```diff
@@ -73,7 +89,16 @@ impl LeafNode {
         let core_node = Node::new(id, capacity, egress_rate, 0, plugin, 0);
         LeafNode { core_node }
     }
+
+    pub fn choose_destination(&self, rpc: &Rpc) -> String {
+        // "respond to every node that has sent a request"
+        if rpc.headers.contains_key("src") {
```

Comment:
We already checked for this, so this is unnecessary.

Reply:
Fixed